wiggum/labeled_dataframe.py

```diff
@@ -214,8 +233,11 @@ def __init__(self,data=None,meta=None,results=None):
 
         # if result_df not empty then load trend_list and type objects
         if len(self.result_df) >0:
-            with open(trends, 'r') as tjson:
-                trend_content = json.load(tjson)
+            if type(trends) is str:
+                with open(trends, 'r') as tjson:
+                    trend_content = json.load(tjson)
+            elif type(trends) is FileStorage:
+                trend_content = json.load(trends)
```

Comment:
why did you not use the safe with open format for this?

Reply:
Because the trends are already a FileStorage type, can't use open function.